config/environments/production.rb

```diff
@@ -66,46 +64,22 @@
   # the I18n.default_locale when a translation cannot be found).
   config.i18n.fallbacks = true
 
-  # Send deprecation notices to registered listeners.
-  config.active_support.deprecation = :notify
+  # Don't log any deprecations.
+  config.active_support.report_deprecations = false
```

Comment:
Why is this turned off?

Reply:
It's the default in rails 7, I think it's because we don't need this in production, you'll look for deprecation warnings in your tests or in development.